Fix JetBrains IDEs identifiers (#2360)
codenvy/che-docs: modules/end-user-guide/examples/snip_che-editor-key-values.adoc

| `eclipse/che-theia/next`
| When using the `next` link:https://github.com/che-incubator/chectl/[chectl] channel, this IDE loads as a default without the URL parameter.

> | link:https://github.com/che-incubator/che-code[Visual Studio Code]

Reviewer: Can we really call it Visual Studio Code?

Suggestion based on the title in https://github.com/microsoft/vscode:

```suggestion
| link:https://github.com/che-incubator/che-code[Visual Studio Code - Open Source]
```

| `che-incubator/che-code/insiders`

> | VS Code Insiders version packaged to run on Kubernetes

Reviewer: 17:3   error    Use 'Visual Studio Code'        RedHat.CaseSensitiveTerms 
                 rather than 'VS Code'.                  

```suggestion
| Visual Studio Code Insiders version packaged to run on Kubernetes
```

| link:https://www.jetbrains.com/help/idea/discover-intellij-idea.html[IntelliJ IDEA]
| `che-incubator/che-idea/latest`
| Community Edition - stable version

| link:https://www.jetbrains.com/help/idea/discover-intellij-idea.html[IntelliJ IDEA]
| `che-incubator/che-idea/next`

> | Community Edition - nigthly version

Reviewer: 25:23  warning  Use correct American English    RedHat.Spelling           
                 spelling. Did you really mean                             
                 'nigthly'?          

```suggestion
| Community Edition - development version
```

| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
| `che-incubator/che-pycharm/latest`
| Community Edition - stable version

| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
| `che-incubator/che-pycharm/next`

> | Community Edition - nightly version

Reviewer: ```suggestion
| Community Edition - development version
```
